python/ray/data/_internal/planner/exchange/aggregate_task_spec.py

```diff
@@ -0,0 +1,111 @@
+from typing import List, Optional, Tuple, Union
+
+from ray.data._internal.planner.exchange.interfaces import ExchangeTaskSpec
+from ray.data._internal.table_block import TableBlockAccessor
+from ray.data.aggregate import _AggregateOnKeyBase, AggregateFn, Count
+from ray.data.block import (
+    Block,
+    BlockAccessor,
+    BlockExecStats,
+    BlockMetadata,
+    KeyFn,
+    KeyType,
+)
+
+
+class SortAggregateTaskSpec(ExchangeTaskSpec):
```

Comment:
This class is mostly copied from [_GroupbyOp](https://github.com/ray-project/ray/blob/master/python/ray/data/grouped_dataset.py#L35).